controls/cis_ocp_1_4_0/section-4.yml

```diff
@@ -12,58 +12,75 @@ controls:
     - id: 4.1.1
       title: Ensure that the kubelet service file permissions are set to 644 or more
         restrictive
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_permissions_worker_service
       level: level_1
     - id: 4.1.2
       title: Ensure that the kubelet service file ownership is set to root:root
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_owner_worker_service
+        - file_groupowner_worker_service
       level: level_1
     - id: 4.1.3
       title: If proxy kube proxy configuration file exists ensure permissions are
         set to 644 or more restrictive
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_permissions_proxy_kubeconfig
       level: level_1
     - id: 4.1.4
       title: If proxy kubeconfig file exists ensure ownership is set to root:root
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_owner_proxy_kubeconfig
+        - file_groupowner_proxy_kubeconfig
       level: level_1
     - id: 4.1.5
       title: Ensure that the --kubeconfig kubelet.conf file permissions are set to
         644 or more restrictive
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_permissions_kubelet_conf
       level: level_1
     - id: 4.1.6
       title: Ensure that the --kubeconfig kubelet.conf file ownership is set to root:root
-      status: pending
-      rules: []
+      status: automated
+      rules:
+        - file_groupowner_kubelet_conf
+        - file_groupowner_kubelet
```

Comment:
This rule doesn't seem to exist?

Reply:
Must have been leftover cruft from the original profile: https://github.com/ComplianceAsCode/content/blob/master/products/ocp4/profiles/cis-node.profile#L144